src/tasks/minions/bso/turaelsTrialsActivity.ts

```diff
@@ -40,6 +40,8 @@ export const turaelsTrialsTask: MinionTask = {
 
 		const { newScore } = await incrementMinigameScore(userID, 'turaels_trials', quantity);
 
+		const name = 'Turaels Trials';
```

Comment:
Dont make a variable like this. Just put the text directly into the string below.

Reply:
okay

Reply:
removed.  that Ellipsis bot is pretty cool,  it new?